### ISSUE TITLE
Unittests / Merge CTL in into scribus main repository ?

### ISSUE THREAD

**Issue by sommerluk** [HOST-Oman/scribus#239]
The file https://github.com/HOST-Oman/scribus/blob/ctl/scribus/plugins/scriptplugin/scripts/LigatursatzUnittests.py didn’t make it in the merge of CTL into the scribus main repository. Is this intentionally?

**Comment by Fahad-Alsaidi** [comment]
@sommerluk I have no control over that. Please fill bug in https://bugs.scribus.net/my_view_page.php

**Comment by luzpaz** [comment]
CC @MrB74

**Comment by luzpaz** [comment]
@sommerluk can you speak more about this test? Is there a way to not make it query the user each time...?

**Comment by sommerluk** [comment]
Yes, I could do that if it’s desired. I guess some of the tests that requiere UI interaction can be changed so that they do not need UI interaction anymore. Some other tests that requiere UI interaction could be dropped…

**Comment by luzpaz** [comment]
CC @gregp-scribus
